flocker/acceptance/testtools.py

```diff
@@ -938,7 +948,8 @@ def check_http_server(host, port):
     """
     req = get(
```

Comment:
:question: Maybe define our own `get` and `post` at the top level that correctly pass timeout & persistent?